.github/scripts/checkParser.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+START_DIR=$(pwd)
+ROOT_DIR=$(dirname "$(dirname "$(dirname "${BASH_SOURCE[0]}")")")
+cd "$ROOT_DIR" || exit 1
+
+autocomplete_parser_backup="src/libs/SearchParser/autocompleteParser.js.bak"
+search_parser_backup="src/libs/SearchParser/searchParser.js.bak"
+
+cp src/libs/SearchParser/autocompleteParser.js "$autocomplete_parser_backup" 2>/dev/null
```

Comment:
NAB: How about adding some comments to this script to better visualise what it does?